cmd/vsphere/main.go

```diff
@@ -138,6 +180,10 @@ func main() {
 		klog.Fatal(err)
 	}
 
+	if err := ipamv1alpha1.AddToScheme(mgr.GetScheme()); err != nil {
```

Comment:
A PR merged earlier to change all `AddToScheme` to `Install`, does ipamv1alpha1 have an `Install` to be consistent?

Reply:
It does not, unfortunately.  I can work up an API update to add that.  Would you prefer we hold this PR until we get that API update in?

Reply:
No it's ok, they are aliases at the end of the day

Reply:
oh, actually, nm, had a cross-wire.  that is an upstream type.